openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -267,12 +289,19 @@ def create_media_popularity_constants_view(
 
 def update_media_popularity_constants(
     postgres_conn_id,
+    task,
```

Comment:
Same comment as above, this will cause an error in the data refresh if we don't update the task to pass all of these in as kwargs.